### ISSUE TITLE
Uncaught Error: Backbone.history has already been started

### ISSUE THREAD

**Issue by zhickson** [bonny/WordPress-Simple-History#319]
**Describe the bug**
This error is thrown when another plugin has already started the Backbone history method. Found this with the Brightcove Plugin specifically. Causes the Simple History screen to be stuck in a loading state. Error is thrown in the browser console.

**To Reproduce**
1. Install and activate the Brightcove Video Connect WordPress Plugin.
2. Activate both Brightcove Connect and Simple History.
3. Go to the Simple History main view.
4. Loading screen will be stuck loading. Error is visible in the browser console.

**Expected behavior**
Simple History scripts.js `manualInitialize` function should probably check whether Backbone history has been started already to avoid compatibility issues.

For example, something like: 
```
if ( ! Backbone.History.started ) {
    Backbone.history.start();
}
```

**System information (please complete the following information):**
 - WordPress version: 6.0.2
 - PHP version: 8.0.1
 - Installed plugins:
+------------------------------------+----------------+-----------+-----------+
| name                               | status         | update    | version   |
+------------------------------------+----------------+-----------+-----------+
| action-scheduler                   | active-network | none      | 3.5.2     |
| acf-repeater                       | active-network | none      | 2.1.0     |
| advanced-custom-fields-pro         | active-network | none      | 6.0.0     |
| allow-numeric-stubs                | active-network | none      | 3.0.0     |
| brightcove-video-connect           | active-network       | none      | 2.7.0     |
| disable-emails                     | active-network | none      | 1.8.1     |
| distributor-stable                 | active-network | none      | 1.6.5-dev |
| jwt-authentication-for-wp-rest-api | active-network | available | 1.2.6     |
| missed-scheduled-posts-publisher   | active-network | none      | 1.0.1     |
| simple-history                     | active-network | none      | 3.3.0     |
| user-role-editor                   | active-network | none      | 4.63.1    |
| user-switching                     | active-network | none      | 1.7.0     |
| wp-crontrol                        | active-network | none      | 1.14.0    |
| wp-statuses                        | active-network | none      | 2.1.3     |
| duplicate-post                     | active-network | none      | 4.5       |
| wordpress-seo                      | active-network | none      | 19.7.1    |

**Comment by bonny** [comment]
Thanks for a good bug report, that event includes a possible solution! Will take a look at this.

**Comment by zhickson** [comment]
@bonny any update on this? we are still having to run a patched edition of ver 3.3.0 due to this bug.

**Comment by bonny** [comment]
@zhickson can you try the just released version https://wordpress.org/plugins/simple-history/#developers and see if that solved the issue?
